unit_tests/kernels/UnitTestScalarAdvDiffElem.C

```diff
@@ -66,14 +62,15 @@ TEST_F(MixtureFractionKernelHex8Mesh, NGP_advection_diffusion)
   // Populate LHS and RHS
   helperObjs.execute();
 
-#ifndef KOKKOS_ENABLE_CUDA
-  EXPECT_EQ(helperObjs.linsys->lhs_.extent(0), 8u);
-  EXPECT_EQ(helperObjs.linsys->lhs_.extent(1), 8u);
-  EXPECT_EQ(helperObjs.linsys->rhs_.extent(0), 8u);
+  Kokkos::deep_copy(helperObjs.linsys->hostlhs_, helperObjs.linsys->lhs_);
```

Comment:
Since we know the sizes of `lhs_` and `rhs_`, would it make sense to consolidate the code from 65-70 within the body of `helperObjs.execute()` and reduce the verbosity in the `TEST_F` body?

Reply:
I agree, I'll do that.